scene/gui/text_edit.cpp

```diff
@@ -1372,13 +1373,17 @@ void TextEdit::_notification(int p_what) {
 
 				if (completion_options.size() < 50) {
 					for (int i = 0; i < completion_options.size(); i++) {
-						int w2 = MIN(cache.font->get_string_size(completion_options[i]).x, cmax_width);
+						int w2 = MIN(cache.font->get_string_size(completion_options[i].display).x, cmax_width);
 						if (w2 > w)
 							w = w2;
 					}
 				} else {
 					w = cmax_width;
 				}
+#ifdef TOOLS_ENABLED
```

Comment:
Won't this make TextEdit behave differently when run via the editor binary but in a game project?